src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -768,6 +785,49 @@ class ArnoldAttributes : public IECoreScenePreview::Renderer::AttributesInterfac
 			{
 				AiNodeSetFlt( node, g_volumePaddingArnoldString, m_volumePadding );
 			}
+
+			if( m_volumeGrids && m_volumeGrids->readable().size() )
+			{
+				AtArray *array = ParameterAlgo::dataToArray( m_volumeGrids.get(), AI_TYPE_STRING );
+				AiNodeSetArray( node, g_volumeGridsArnoldString, array );
+			}
+
+			if( m_velocityGrids && m_velocityGrids->readable().size() )
+			{
+				AtArray *array = ParameterAlgo::dataToArray( m_velocityGrids.get(), AI_TYPE_STRING );
+				AiNodeSetArray( node, g_velocityGridsArnoldString, array );
+			}
+
+			if( m_velocityScale != 1.0f && AiNodeEntryLookUpParameter( AiNodeGetNodeEntry( node ), g_velocityScaleArnoldString ) )
+			{
+				AiNodeSetFlt( node, g_velocityScaleArnoldString, m_velocityScale );
+			}
+
+			if( m_velocityScale > 0 )
+			{
+				AtNode *options = AiUniverseGetOptions();
+				const AtNode *arnoldCamera = static_cast<const AtNode *>( AiNodeGetPtr( options, "camera" ) );  // \todo: option name is declared below
+
+				if( arnoldCamera )
+				{
+					float shutterStart = AiNodeGetFlt( arnoldCamera, g_shutterStartArnoldString );
+					float shutterEnd = AiNodeGetFlt( arnoldCamera, g_shutterEndArnoldString );
+
+					AiNodeSetFlt( node, g_motionStartArnoldString, shutterStart );
```

Comment:
This is trying to set the motion_start and motion_end parameters on _any_ node, not just volume nodes.